python/setup.py

```diff
@@ -225,6 +225,8 @@ def get_packages(self):
             pandas_dep,
             "pyarrow >= 6.0.1, < 7.0.0",
             "fsspec",
+            "pymongo",
+            "pymongoarrow",
```

Comment:
we shouldn't actually depend on mongo here right? this should only be an extra just in case users want to install it?

Reply:
Agreed. This should not be a required dependency.

Reply:
no, so for example, we have from_spark but we do not depend on it here.

the set listed here should be the minimal set to use basic, non-integration datasets functionality.